ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -231,7 +231,7 @@ import '../nuxeo-button-styles.js';
             <nuxeo-data-table-row header>
               <nuxeo-data-table-checkbox
                 style$="[[_computeSelectAllVisibility(selectionEnabled, selectAllEnabled, multiSelection)]]"
-                checked="[[selectAllActive]]"
+                checked="[[_isChecked(selectAllActive, _excludedItems, _excludedItems.*)]]"
```

Comment:
@Gabez0r  comment on the previous PR.
https://github.com/nuxeo/nuxeo-elements/pull/602#pullrequestreview-1103956476